aep/general/0003/aep.md

```diff
@@ -0,0 +1,112 @@
+# Glossary
+
+This AEP defines common terminology.
+
+## Guidance
+
+The following terminology **must** be used consistently throughout AEPs.
+
+### API
+
+Application Programming Interface. This can be a local interface (such as an
+SDK) or a Network API (defined below).
+
+### API Backend
+
+A set of servers and related infrastructure that implements the business logic
+for an API Service.
+
+### API Client
+
+An API Client is a program or library that perform a specific tasks by calling
+an API or generic tools, such as CLIs, that expose the API in a user-accessible
+fashion or operate on resource data at rest.
+
+Examples of clients include the following:
+
+- Command line interfaces
+- Libraries, such as an SDK for a particular programming language
+- Scripts that operates on a JSON representation of a resource after reading it
+  from an API
+- Tools, such as a [Declarative client][]
+- Visual UIs, such as a web application
+
+### API Definition
+
+A well-structured representation of an API Service.
+
+### API Endpoint
+
+Refers to a network address that an API Service uses to handle incoming API
+Requests. One API Service may have multiple API Service Endpoints, such as
+`https://pubsub.googleapis.com` and `https://content-pubsub.googleapis.com`.
+
+### API Gateway
+
+One or more services that together provide common functionality across API
+Services, such as load balancing and authentication.
+
+### API Method
+
+An individual operation within an API. It is typically represented in Protocol
+Buffers by an `rpc` definition, or in HTTP via a `method` and a `path`.
+
+### API Request
+
+A single invocation of an API Method. It is often used as the unit for billing,
+logging, monitoring, and rate limiting.
+
+### API Resource
+
+An object that one or more API methods operate on.
```

Comment:
Nit:  this sentence ends with a preposition.  I am ambivalent about this, since avoiding doing so is a kind of archaic rule, but noting it nevertheless.

Reply:
I'm less opinionated about grammar, but fine with `An object upon which one or more API methods operate`. WDYT? Frankly my knowledge of english grammar is quite poor.

Reply:
Updated.